extensions/ql-vscode/README.md

```diff
@@ -28,7 +28,7 @@ For information about other configurations, see the separate [CodeQL help](https
 1. [Run a query](#running-a-query).
 
 ---
-
+<!-- markdownlint-disable-next-line MD024 -->
```

Comment:
MD024 is [multiple headings with the same content](https://github.com/DavidAnson/markdownlint/blob/main/doc/md024.md). It seems okay here as you're providing a short and long version of the same steps.